answer_unit.php

```diff
@@ -66,20 +73,61 @@ public function __construct() {
         $this->default_mapping = null;
         $this->mapping = null;
         $this->additional_rules = '';
+        $this->part_unit = '';
     }
 
+    /**
+     * Parse a unit into prefix and SI unit
+     * 
+     * @param string $unit get the unit with prefix and SI unit, must be trimmed before
+     * @return string SI unit only
+     */
+    public function parse_prefix_unit($unit) {
+        if (is_array($unit)) {
+            throw new Exception('parse_prefix_unit does not handle arrays!');
```

Comment:
The string should be in `lang/en/qtype_formulas.php` and imported with `get_string`.